pkgs/development/interpreters/perl/default.nix

```diff
@@ -159,6 +159,8 @@ let
         # into $dev/lib/perl5/cross_perl/${version}.
         wrapProgram $dev/bin/perl --prefix PERL5LIB : \
           "$dev/lib/perl5/cross_perl/${version}:$out/lib/perl5/${version}:$out/lib/perl5/${version}/$runtimeArch"
+
+        $STRIP $dev/bin/perl
```

Comment:
Is not this the wrapper? Also why is not it stripped automatically?

Reply:
right that would be a no-op